src/language-client.ts

```diff
@@ -70,7 +72,6 @@ export class LanguageClient implements Disposable {
   }))
 
   private currentDocuments = new Map<string, TextDocument>()
-  // private currentDocumentMasters = new Map<string, TextDocuments<TextDocument>>()
```

Comment:
do we want to keep this comment ?

Reply:
No, why wouldn't we?

Reply:
Then we should remove it ;)

Reply:
it is!?

Reply:
Damn I don't know how to read a diff anymore